app/src/main/java/com/example/workoutapp/WorkoutActivity.kt

```diff
@@ -186,44 +113,150 @@ fun MyWorkout(
                     }
                 )
 
+
             },
+
             content = {
+                Column(
+                    modifier = modifier
+                        .padding(16.dp)
+                        .fillMaxWidth(),
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    Text(
+                        text = "My Workout Week",
+                        style = MaterialTheme.typography.h4,
+                        fontWeight = FontWeight.Bold,
+                        modifier = Modifier.padding(bottom = 8.dp),
+                        textAlign = TextAlign.Center
+                    )
+                    Spacer(modifier = Modifier.height(20.dp))
+
+                    for (myWorkoutDay in myWorkoutWeek) {
+                        MyWorkoutExpandableItem(myWorkoutDay)
+                        Spacer(modifier = Modifier.height(20.dp))
+                    }
+                    Spacer(modifier = Modifier.height(20.dp))
+
+                    IconButton(
+                        onClick = {
+                        })
+                    {
+                        Icon(imageVector = Icons.Filled.AddCircle,
+                            contentDescription = null,
+                            modifier = Modifier.size(50.dp),
+                            tint = Teal200)
+                    }
+                }
             }
         )
 
+
     }
+}
+
+/* Composable function MyWorkoutExpandableItem that displays a single workout item in a Column layout. The function takes in
+   a MyWorkoutDay data class object as a parameter, which contains the Day and a list of exercises. */
+
+/* The workout's name is displayed using a Text composable function within a Row layout, which also
+   contains an Icon that toggles the display of the exercises when clicked. */
 
+/* If the user clicks on a row, the "expanded" variable will be set to true, and the MyExerciseItem
+   composable function will be called to display the list of exercises. */
```

Comment:
Use proper KDoc format above a function.
https://medium.com/@drflakelorenzgerman/documentation-with-kdoc-for-kotlin-android-a93c99dfe74